.github/workflows/tests.yml

```
@@ -0,0 +1,68 @@
+
+      - name: Generate app key
+        run: php artisan key:generate
+
+      - name: Make migrations
+        run: php artisan migrate
+
+      - name: Seed the database
+        run: php artisan db:seed
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v2
```

actionlint

the runner of "actions/setup-node@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)